CMakeLists.txt

```diff
@@ -1308,6 +1308,7 @@ if(CONFIG_GEN_ISR_TABLES)
     --kernel $<TARGET_FILE:${ZEPHYR_LINK_STAGE_EXECUTABLE}>
     --intlist-section .intList
     --intlist-section intList
+    $<$<BOOL:${CONFIG_ISR_TABLE_USE_INTERRUPT_ID}>:--max-interrupt-id=${CONFIG_INTERRUPT_ID_COUNT}>
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/75946#issuecomment-2246678467

Following your suggestion, I forked gen_isr_tables.py.
Accordingly, the device tree is now processed within gen_isr_tables, and options are no longer necessary.
I think this will make it easier to extend it for RA2AL.